packages/gestalt-datepicker/src/DatePicker.js

```diff
@@ -38,9 +39,26 @@ type Props = {|
   rangeEndDate?: Date,
   rangeSelector?: 'start' | 'end',
   rangeStartDate?: Date,
+  timezone?: Timezone,
   value?: Date,
 |};
 
+export const getDisplayDate = (date: Date, timezone: Timezone): Date => {
+  if (timezone === 'utc') {
+    return new Date(date.getUTCFullYear(), date.getUTCMonth(), date.getUTCDate());
```

Comment:
can we add here a comment?

// react-datepicker only supports local timezones. When 'value' prop is a UTC date, it must be turned into local  timezone before passing it to react-datepicker